x-pack/plugins/apm/server/lib/rum_client/has_rum_data.ts

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Maybe you could move this file to `server/lib/observability_overview`

Reply:
for code ownership purpose all the ux api are in rum_client folder so it made sense for this as well. This code is going out of APM in 7.11/7.12 so keeping it together make sense.